Makefile

```
@@ -549,6 +549,10 @@ endif
 ui-test:
 	make -C ui test
 
+.PHONY: ui-test-component
+ui-test-component:
+	make -C ui test-component
```

To match the job name.
```suggestion
.PHONY: ui-component-tests
ui-component-tests:
	make -C ui test-component
```